### ISSUE TITLE
Consider whether missing member lookup on tuple should complain about tuple type or the underlying.

### ISSUE THREAD

**Issue by VSadov** [dotnet/roslyn#10874]
Currently code like 

``` C#
        var x0 = new (int a, int b)(1, 2, 3);
```

results in 
                // (6,22): error CS1729: 'ValueTuple<int, int>' does not contain a constructor that takes 3 arguments
                //         var x0 = new (int a, int b)(1, 2, 3);

The error is technically correct - it is the ValueTuple whose constructor is missing. Actual tuple only has two fields - "a" and "b".

However, is this a right model from the user point of view? 
Should we complain about `(int a, int b)` missing a constructor?

**Comment by jcouv** [comment]
This probably doesn't make sense after the tuple type unification. Can you confirm?

**Comment by VSadov** [comment]
The sample is no longer relevant since it is no longer a valid syntax.

In other examples we refer to tuple, not to underlying type:

```c#
        var x0 = (1, 2, 3).Foo;
```

```
error CS1061: '(int, int, int)' does not contain a definition for 'Foo' and no extension method 'Foo' accepting a first argument of type '(int, int, int)' could be found (are you missing a using directive or an assembly reference?)
```
